service.subtitles.pipocas/service.py

```diff
@@ -223,6 +271,7 @@ def append_subtitle(item):
     xbmcplugin.addDirectoryItem(handle=int(sys.argv[1]), url=url, listitem=listitem, isFolder=False)
 
 def Search(item):
+    enable_rar()
```

Comment:
You probably want to restrict this to the kodi version that introduced vfs.rar (18?). The addon is in gotham branch, which means any users of gotham-leia will receive the update

Reply:
Yeah, you are right!
Should I re-submit?

Reply:
I am always in favor or dropping support for legacy/unmaintained stuff. So if I was in your position I'd mark the gotham addon as broken and move this to an upper repo, let's say krypton or even leia.

Note: When matrix is out, it will only support python3 and there'll be a breaking change on our python api

Reply:
I know Enen, that is why I don't want to spend time around this script. I want to focus on "upgrading" to python3.
Meanwhile yesterday I fixed a few things you requested, maybe you should cancel this pull and I will do a new one.
Thanks.

Reply:
Python3 will require a new PR to matrix unfortunately. You won't be able to use the same "version" in both python2 (gotham-leia) and python3 (matrix-) due to the hard dependency on `xbmc.python` v3.0.0. We are taking this opportunity to cleanup broken addons in the repo and also cleanup the python API which has like 7 years of backward compatibility and non-sense :)